### ISSUE TITLE
Nothing happens

### ISSUE THREAD

**Issue by dany68** [jkroso/atom-browser-refresh-on-save#3]
Hi,
Thanks for your work... but your package doesn't work on my mac. Nothing happens. When I press cmd-s, atom just save my files.
There is any configuration to do ?

**Comment by dany68** [comment]
I find the problem. I use CakePHP so my pages have for extension .ctp and no .html.
To solve this:
- Go to plugin.coffee
- Add scripts['ctp'] = (after html)
- Restart atom
- Enjoy

Please can you add ctp extension in your next update ?

**Comment by jkroso** [comment]
I think I need to add some config options

**Comment by dany68** [comment]
Yes, like I said you need to add ctp support in plugin.coffee file (for cakephp users).
For the moment I add it, but because it's not a default integration each time I will update the package, I will need to config it manualy.
